### PR TITLE
RNN long-term dependencies with auxiliary losses

### PR DESCRIPTION
**Pull request by ppoulin91**

### CODE REVIEW

File: deep-learning/_posts/2018-07-05-rnn-long-term.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+author: "Philippe Poulin"
+cite:
+    authors: "H. Trinh, T., M. Dai, A., Minh-Thang, L., Quoc V., L."
+    title:   "Learning Longer-term Dependencies in RNNs with Auxiliary Losses"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/pdf/1803.00144"
+---
+
+**TL;DR**:
+For a sequence classification task using an RNN, adding small reconstruction/prediction tasks throughout the sequence reduces the need for full BPTT and increases performance. Even when using only the auxiliary loss (no supervised loss backprop in the LSTM), results are "good" (40#% accuracy on CIFAR-10).
```
Comment (CharlesAuthier) [inline deep-learning/_posts/2018-07-05-rnn-long-term.md]: '40#%' ?

Comment (ppoulin91) [reply deep-learning/_posts/2018-07-05-rnn-long-term.md]: Damn, typo...

File: deep-learning/_posts/2018-07-05-rnn-long-term.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+---
+layout: review
+title: "Learning Longer-term Dependencies in RNNs with Auxiliary Losses"
+tags: deep-learning RNN
```
Comment (jodoin) [inline deep-learning/_posts/2018-07-05-rnn-long-term.md]: classification

File: deep-learning/_posts/2018-07-05-rnn-long-term.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+![](/deep-learning/images/rnn-long-term/figure4.png)
+
+
+- Training/testing curves:
+
+![](/deep-learning/images/rnn-long-term/figure5.png)
+
+### Notes
+
+- The authors never used both auxiliary losses at the same time, whoch would have been interesting
```
Comment (jodoin) [inline deep-learning/_posts/2018-07-05-rnn-long-term.md]: whoch?

**Pull request merged**